package.json

```diff
@@ -31,6 +31,8 @@
     "request": "^2.85.0",
     "request-promise": "^4.2.2",
     "rwlock": "^5.0.0",
+    "sqlite": "^2.9.2",
+    "sqlite3": "^4.0.0",
```

Comment:
are we using both?

Reply:
Yeah `sqlite` is a promisified wrapper of `sqlite3`, and `sqlite3` is the underlying binding to the native version.

Reply:
And we need `sqlite3`'s API in some places too because `sqlite` doesn't wrap everything